libsolidity/ast/Types.cpp

```diff
@@ -2436,6 +2436,11 @@ MemberList::MemberMap FunctionType::nativeMembers(ContractDefinition const*) con
 	case Kind::BareDelegateCall:
 	{
 		MemberList::MemberMap members;
+		if (m_kind == Kind::External && hasDeclaration())
```

Comment:
Didn't we say that the `hasDeclaration()` condition is not required?

Reply:
Need to give it a try.